analysis/compare_nino_spectra_fig31/compare_nino_spectra.jnl

```diff
@@ -0,0 +1,68 @@
+\can mode verify
+! Usage: go compare_nino_spectra [idx idx_title hspec print_opt rebuild]
+!                                [ $1     $2      $3      $4       $5  ]
+
+def sym idx "$1%nino3%"
+def sym idx_title "$2%NINO3%"
+def sym hspec $3"0:3.5:.5"
+def sym print_opt $4"0"
+let rebuild = $5"0"
+
+can dat/all
+use "/home/atw/data/reynolds/v5/noaa.ersst/sst.nc"
```

Comment:
@aradhakrishnanGFDL here is someone's home directory.  This will only work at GFDL